packages/flutter/test/material/selection_area_test.dart

```diff
@@ -361,7 +361,7 @@ void main() {
       // After lifting the finger up, the selection toolbar should be showing again.
       expect(find.text('Copy'), findsOneWidget);
     },
-    variant: TargetPlatformVariant.all(),
+    variant: TargetPlatformVariant.mobile(),
```

Comment:
Can you explain why this is changed?

Reply:
+1, maybe just don't do this in this PR. I guess you did it just because this test appears to be relevant only to mobile? But I worry that it could conceal a failure if we make this change in this PR.